# file: C:\Users\Lisardo Iniesta\AppData\Roaming\Python\Python313\site-packages\_pytest\_py\error.py
# hypothesis_version: 6.142.1

[267, 'P', 'R', 'UnknownErrnoNone', '[{}]: {}', '_', '__doc__', '__module__', 'errno', 'py.error', 'win32', '{}.{} {!r}: {} ']